### ISSUE TITLE
volo on object detection with detectron2?

### ISSUE THREAD

**Issue by jinfagang** [sail-sg/volo#3]
volo on object detection with detectron2?

**Comment by yuanli2333** [comment]
Hi, volo on other downstream tasks is on the way. 
We will release more results in the future.

**Comment by jinfagang** [comment]
@yuanli2333  Hopefully you guys using detectron2 rather than mmdetection.

**Comment by yuanli2333** [comment]
Is there any speical reason to use detectron2 rather than mmdet?

**Comment by jinfagang** [comment]
No, I just think d2 could be simpler to follow, haha. mmdet more academic though.

**Comment by ld-xy** [comment]
大佬们，有没有tensorflow实现的模型代码？？？(Do you have any model code for TensorFlow implementation??)
